js/src/indexation.js

```diff
@@ -122,7 +122,15 @@ class Indexing extends Component {
 				"X-WP-Nonce": nonce,
 			},
 		} );
-		return response.json();
+
+		const data = await response.json();
+
+		// Throw an error when the response's status code is not in the 200-299 range.
+		if ( ! response.ok ) {
+			throw new Error( data.message );
```

Comment:
Just a question: Are we catching this somewhere?

Reply:
We catch this in the `doIndexing` method, here in `indexation.js` 🙂.